c++/src/OrcFile.cc

```diff
@@ -161,6 +163,7 @@ DIAGNOSTIC_POP
     }
 
     void write(const void* buf, size_t length) override {
+      SCOPED_STOPWATCH(metrics, IOBlockingLatencyUs, IOCount);
```

Comment:
This may not be the right place. We should measure where it is called. Otherwise we cannot support HDFS and other file systems.

Reply:
To support all file systems, the macro SCOPED_STOPWATCH for IO metrics is place in class BufferedOutputStream(OutputStream.cc).